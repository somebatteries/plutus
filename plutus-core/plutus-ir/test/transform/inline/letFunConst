(let
  (nonrec)
  (datatypebind
    (datatype
      (tyvardecl Bool (type))

      Bool_match
      (vardecl True Bool) (vardecl False Bool)
    )
  )
  (termbind
    (strict)
    (vardecl constFun (fun (con integer) (fun Bool (con integer))))
    (lam x (con integer) (lam y Bool x))
  )
  [ [ constFun (con integer 3) ] False ]
)
