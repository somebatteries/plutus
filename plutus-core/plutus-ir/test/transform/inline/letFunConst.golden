ParseErrorB (ParseErrorBundle {bundleErrors = TrivialError 1 (Just (Tokens ('p' :| "rogram"))) (fromList [Tokens ('a' :| "bs"),Tokens ('b' :| "uiltin"),Tokens ('c' :| "on"),Tokens ('e' :| "rror"),Tokens ('i' :| "wrap"),Tokens ('l' :| "am"),Tokens ('l' :| "et"),Tokens ('u' :| "nwrap")]) :| [], bundlePosState = PosState {pstateInput = "(program\n  (let\n    (nonrec)\n    (datatypebind\n      (datatype\n        (tyvardecl Bool (type))\n\n        Bool_match\n        (vardecl True Bool) (vardecl False Bool)\n      )\n    )\n    (termbind\n      (strict)\n      (vardecl constFun (fun (con integer) (fun Bool (con integer))))\n      (lam x (con integer) (lam y Bool x))\n    )\n    [ [ constFun (con integer 3) ] False ]\n  )\n)", pstateOffset = 0, pstateSourcePos = SourcePos {sourceName = "letFunConst", sourceLine = Pos 1, sourceColumn = Pos 1}, pstateTabWidth = Pos 8, pstateLinePrefix = ""}})